src/listener/Lists.php

```diff
@@ -49,7 +49,7 @@ public function render(Lexer $lexer)
                 // its the same line as the start.. skip this one as its by default included in while operations
                 if ($line == $pick->line) {
                     return true;
-                } elseif (($line->hasEndNewline() || $line->hasNewline())) {
+                } elseif (($line->hasEndNewline() || $line->hasNewline() || $line->isJsonInsert())) {
```

Comment:
I've also been thinking about doing:

```suggestion
                } elseif (($line->hasEndNewline() || $line->hasNewline() || ($line->isJsonInsert() && $line->isInline() === false))) {
```

As an extra safe guard. But I don't know if that is needed?

Reply:
Yes good idea. Maybe use !$line->isInline() instead of compare against false.